ballerina-tests/tests/http2_to_http1_client_fallback_test.bal

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.package http2;
+
+import ballerina/http;
+import ballerina/test;
+
+listener http:Listener generalSSLListener = new(httpSslGeneralPort, {
+    httpVersion: "1.1",
+    secureSocket: {
+        key: {
+            path: "tests/certsandkeys/ballerinaKeystore.p12",
+            password: "ballerina"
+        }
+    }
+});
+
+final http:Client fallbackClient = check new("http://localhost:" + generalPort.toString(), httpVersion = "2.0");
```

Comment:
Since the default client is H2, we don't need to specify the version here
```suggestion
final http:Client fallbackClient = check new("http://localhost:" + generalPort.toString());
```